apps/dokploy/public/locales/en/settings.json

```diff
@@ -0,0 +1,68 @@
+{
+	"settings": {
+		"common": {
+			"save": "Save"
+		},
+		"server": {
+			"domain": {
+				"title": "Server Domain",
+				"description": "Add a domain to your server application.",
+				"form": {
+					"domain": "Domain",
+					"letsEncryptEmail": "Let's Encrypt Email",
+					"certificate": {
+						"label": "Certificate",
+						"placeholder": "Select a certificate"
+					},
+					"certificateOptions": {
+						"none": "None",
+						"letsencrypt": "Let's Encrypt (Default)"
+					}
+				}
+			},
+			"webServer": {
+				"title": "Web Server",
+				"description": "Reload or clean the web server.",
+				"actions": "Actions",
+				"reload": "Reload",
+				"watchLogs": "Watch logs",
+				"server": {
+					"label": "Server"
+				},
+				"traefik": {
+					"label": "Traefik",
+					"modifyEnv": "Modify Env"
+				},
+				"storage": {
+					"label": "Space",
+					"cleanUnusedImages": "Clean unused images",
+					"cleanUnusedVolumes": "Clean unused volumes",
+					"cleanStoppedContainers": "Clean stopped containers",
+					"cleanDockerBuilder": "Clean Docker Builder & System",
+					"cleanMonitoring": "Clean Monitoring",
+					"cleanAll": "Clean all"
+				}
+			}
+		},
+		"profile": {
+			"title": "Account",
+			"description": "Change the details of your profile here.",
+			"email": "Email",
+			"password": "Password",
+			"avatar": "Avatar"
+		},
+		"appearance": {
+			"title": "Appearance",
+			"description": "Customize the theme of your dashboard.",
+			"theme": "Theme",
+			"themeDescription": "Select a theme for your dashboard",
+			"themes": {
+				"light": "Light",
+				"dark": "Dark",
+				"system": "System"
+			},
+			"language": "Language",
+			"languageDescription": "Select a language for your dashboard"
+		}
+	}
+}
```

Comment:
can we change this way to write the keys? initially is very easy to understand the semantic, however when there are alot of keys it can be very complex and more dificult to find a specific key, I would like this way

`settings.common.save` : "Save"

is more easy to search

Reply:
Done!